### PR TITLE
feat(analytics): add createAccount()

### PR DESCRIPTION
**Pull request by sabmoonie**
Add `createAccount()`. This will save user data into Mixpanel.

### CODE REVIEW

File: packages/analytics/tests/analytics.spec.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { ObjectId } from '@highoutput/object-id';
+import Chance from 'chance';
+
+describe('analytics', () => {
+  const chanceHelper = new Chance();
+  it('should create an account', async () => {
+    const mockAnalytics = {
```
Comment (rogermadjos) [inline packages/analytics/tests/analytics.spec.ts:7]: you should be mocking the mixpanel client

File: packages/analytics/src/index.ts [hunk 2]
```diff
@@ -1 +1,32 @@
...
+export class Analytic {
+  protected project: string;
+  protected driver: Mixpanel;
+  constructor(params: { project: string }) {
+    this.project = params.project;
+    this.driver = mixpanel.init(process.env.MIXPANEL_TOKEN || 'secrets');
+  }
+
+  createAccount(params: {
+    accountId: ObjectId;
```
Comment (rogermadjos) [inline packages/analytics/src/index.ts:13]: I think this should be a string

**Review by rogermadjos — CHANGES_REQUESTED** [review]

File: packages/analytics/src/index.ts [hunk 3]
```diff
@@ -1 +1,32 @@
...
+    firstname?: string;
+    lastname?: string;
+    email: string;
+    organizationId?: ObjectId;
```
Comment (ccpacillos) [inline packages/analytics/src/index.ts:17]: I don't think `organiationId` needs to be here. Let's remove this.

**Review by ccpacillos — CHANGES_REQUESTED** [review]

File: packages/analytics/tests/analytics.spec.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+function setup(params: { project: string; mockedMixpanelInstance: unknown }) {
+  (mixpanel as jest.Mocked<typeof mixpanel>).init.mockImplementation(
+    () => params.mockedMixpanelInstance as unknown as Mixpanel,
+  );
+  const analytics = new Analytics({ project: params.project });
+
+  return { analytics };
+}
+
+describe('analytics', () => {
```
Comment (ccpacillos) [inline packages/analytics/tests/analytics.spec.ts:18]: Kindly change this to `'Analytics'`.

File: packages/analytics/tests/analytics.spec.ts [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  const chanceHelper = new Chance();
+  test('should create an account', async () => {
```
Comment (ccpacillos) [inline packages/analytics/tests/analytics.spec.ts:20]: Add a second layer of `describe('#createAccount', () => {})`.

**Review by ccpacillos — CHANGES_REQUESTED** [review]

**Review by ccpacillos — APPROVED** [review]
Looks good to me.

**Pull request merged**